.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -34,3 +33,6 @@ Start below this comment section.
 
 
 **Any open or closed issues related to this feature request?**
+
+**Will you be able to contribute a Pull Request for this issue?**  
+Help is always welcome, is this bug something you or your organization plan to fix ?
```

Comment:
```suggestion
**Will you be able to contribute a pull request for this issue?**  
Help is always welcome, is this bug something you or your organization plan to fix?
```

Reply:
changed with the last comment's rephrasing